.github/workflows/add-log.yml

```diff
@@ -0,0 +1,37 @@
+on: [push]
+
+jobs:
+  sequence_and_integrate_job:
+    runs-on: ubuntu-20.04
+    name: Sequence and integrate pending log entries
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Prerequisites
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y golang
+        go build -o /tmp/integrate ${GITHUB_WORKSPACE}/cmd/integrate
+        go build -o /tmp/sequence ${GITHUB_WORKSPACE}/cmd/sequence
+
+    - name: Sequence
+      run: |
+        SERVERLESS_LOG_PUBLIC_KEY=${{ secrets.SERVERLESS_LOG_PUBLIC_KEY }} \
+        /tmp/sequence --storage_dir=${GITHUB_WORKSPACE}/log --logtostderr --entries ${GITHUB_WORKSPACE}/log/leaves/pending/*
+
+    - name: Integrate
+      run: |
+        SERVERLESS_LOG_PRIVATE_KEY=${{ secrets.SERVERLESS_LOG_PRIVATE_KEY }} \
+        SERVERLESS_LOG_PUBLIC_KEY=${{ secrets.SERVERLESS_LOG_PUBLIC_KEY }} \
+        /tmp/integrate --storage_dir=${GITHUB_WORKSPACE}/log --logtostderr
+
+    - name: Clean
+      run: |
+        rm ${GITHUB_WORKSPACE}/log/leaves/pending/*
+
```

Comment:
For now, can just use the actions already provided in the [serverless repo](https://github.com/google/trillian-examples/tree/master/serverless/deploy/github/sequence_and_integrate)?

```yaml
- name: Sequence and integrate step
      id: sequence_and_integrate
      uses: google/trillian-examples/serverless/deploy/github/sequence_and_integrate@master
      with:
        log_dir: './log'
      env:
        SERVERLESS_LOG_PRIVATE_KEY: ${{ secrets.SERVERLESS_LOG_PRIVATE_KEY }}
        SERVERLESS_LOG_PUBLIC_KEY: ${{ secrets.SERVERLESS_LOG_PUBLIC_KEY }}
```

If you want, you can change the `@master` to pin the action to a commit you've inspected and are happy with.
(At some point I'd like for us to move the serverless stuff and actions out into their own repos and we can make proper releases of each of them.)